clinic/clinic-user-guide-for-tiup.md

```diff
@@ -0,0 +1,341 @@
+
+- Methods 1: If the network where the cluster is located can directly connect to the Clinic Server, you can [directly upload data using the upload command](#method-1-upload-directly).
+- Methods 2: If the network where the cluster is located cannot directly connect to the Clinic Server, you need to [pack the data and then upload it](#method-2-pack-and-upload-data).
+
+#### Method 1: Upload directly
+
+When the network where the cluster is located can directly connect to the Clinic Server, you can directly upload the folder with collected data obtained in [Step 2: Collect data](#step-2-collect-data) using the following command:
+
+{{< copyable "shell-regular" >}}
+
+```bash
+ tiup diag upload ${filepath} -u=username -p='password'
+ ```
+
+> **Note:**
+>
+> Clinic is currently in the Beta testing stage, so only the invited users can use the service. You need to get a trial account from the PingCAP technical support staff you contacted before.
+
+The output can be as follows:
+
+{{< copyable "shell-regular" >}}
+
+```bash
+[root@Copy-of-VM-EE-CentOS76-v1 qiaodan]# tiup diag upload /home/qiaodan/diag-fNTnz5MGhr6
+Starting component `diag`: /root/.tiup/components/diag/v0.5.1/diag upload /home/qiaodan/diag-fNTnz5MGhr6
+Enter Username: username
+Enter Password: >>>>>>>>>>>>>>>>>>>>>>>>>>>>>><>>>>>>>>>
+Completed!
+Download URL: "https://clinic.pingcap.com:4433/diag/files?uuid=XXXX"
+```
+
+After the upload is complete, you need to send the data access link of `Download URL` to the PingCAP technical support staff you contacted before.
+
+> **Note:**
+>
+> In the Beta version of the Clinic diagnostic service, external users cannot use the features of the Clinic server. The data access link is only open for the PingCAP technical support staff.
+
+#### Method 2: Pack and upload data
+
+If your cluster is deployed offline, you need to pack the data on your intranet and upload it to the Clinic Server using a network-connected device, because the network where the cluster is located cannot directly connect to the Clinic Server. The detailed operations are as follows:
+
+1.Compress and encrypt the data package collected in [Step 2: Collect data](#step-2-collect-data).
+
+    {{< copyable "shell-regular" >}}
+
+    ```bash
+    tiup diag package ${filepath}
+    ```
+
+    When packaging, Diag encrypts and compresses the data at the same time. In the test environment, 800 MB of data was compressed to 57 MB. The output can be as follows:
+
+    ```bash
+    Starting component `diag`: /root/.tiup/components/diag/v0.5.1/diag package diag-fNTnz5MGhr6
+    packaged data set saved to /home/qiaodan/diag-fNTnz5MGhr6.diag
+    ```
+
+    After the data is packaged through the `package` command above, the data is in the `.diag` format. This file can only be decrypted and viewed after uploading it to the Clinic Server. If you need to directly forward the collected data instead of viewing it in the Clinic Server, you can compress the data and forward it by yourself.
```

Comment:
```suggestion
    After the data is packaged through the `package` command above, the data is in the `.diag` format. This file can only be decrypted and viewed after uploading it to the Clinic Server. If you need to directly forward the collected data instead of viewing it in the Clinic Server, you can compress the data by your own method and forward it.
```